### PR TITLE
Fix extended tests

### PR DESCRIPTION
**Pull request by mlaszloffy**

### CODE REVIEW

File: seqware-archetypes/seqware-archetype-java-workflow/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -246,9 +246,9 @@
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <skip>${skipITs}</skip>
-                            <executable>${workflow-dir}/bin/jre1.6.0_29/bin/java</executable>
-                            <commandlineArgs>-jar ${workflow-dir}/lib/seqware-distribution-${seqware-version}-full.jar --plugin net.sourceforge.seqware.pipeline.plugins.BundleManager -- -t -b ${project.build.directory}/${workflow-full-name}/ </commandlineArgs>
+                            <skip>\${skipITs}</skip>
+                            <executable>\${workflow-dir}/bin/jre1.6.0_29/bin/java</executable>
```
Comment (morgantaschuk) [inline seqware-archetypes/seqware-archetype-java-workflow/src/main/resources/archetype-resources/pom.xml]: Don't we need Java 8? Or at least 7?

File: seqware-ext-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/PluginRunnerET.java [hunk 2]
```diff
@@ -211,6 +215,51 @@ private static void createSharedTempDir() {
...
+            String listCommand = "-p net.sourceforge.seqware.pipeline.plugins.BundleManager -- -l -b " + bundleDir.getAbsolutePath();
+            String listOutput = ITUtility.runSeqWareJar(listCommand, ReturnValue.SUCCESS, null);
+            LOGGER.info(listOutput);
+        }
+    }
+
+    @Test
+    public void testGetWorkflowIniParameters() throws IOException {
+        // this assumes all installed workflows have the same ini - which is true as the only archetype being tested is "java-workflow"
+        String expectedIni = "#key=cat:type=text:display=F\n"
```
Comment (morgantaschuk) [inline seqware-ext-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/PluginRunnerET.java]: I think Andre ripped out all the code that parses comments in INI files, so we should remove them in this test.

File: seqware-archetypes/seqware-archetype-java-workflow/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (mlaszloffy) [reply seqware-archetypes/seqware-archetype-java-workflow/src/main/resources/archetype-resources/pom.xml]: Yes, java 8 is required.  The workflow archetype specifies jre-8.0.45 as a dependency, I'll update this maven goal to use jre-1.8.0_45/bin/java.

File: seqware-ext-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/PluginRunnerET.java (hunk 2 above)
Comment (mlaszloffy) [reply seqware-ext-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/PluginRunnerET.java]: The workflow param endpoint doesn't return key + type + display info/comments, but the command line "list-workflow-params" still returns comments: https://github.com/oicr-gsi/niassa/blob/7b5043ed431fe9074622eb7f65959d4e18ee716d/seqware-pipeline/src/main/java/net/sourceforge/seqware/pipeline/plugins/BundleManager.java#L298

**Pull request merged**